share/dkg/dkg.go

```diff
@@ -815,7 +834,8 @@ func (d *DistKeyGenerator) ProcessJustifications(bundles []*JustificationBundle)
 		// that should not happen in the threat model but we still returns the
 		// fatal error here so DKG do not finish
 		d.state = FinishPhase
-		return nil, fmt.Errorf("process-justifications: only %d/%d valid deals - dkg abort", allGood, targetThreshold)
+		err = fmt.Errorf("process-justifications: only %d/%d valid deals - dkg abort", allGood, targetThreshold)
+		return
```

Comment:
at this point can both res and err have a value then?

Reply:
the result is only computed at the end line 841 `d.computeResult()`- given err is `*Result` it's a nil pointer at this line so far.